config/CLIConfiguration.ts

```diff
@@ -405,25 +408,23 @@ class CLIConfiguration {
     safelyApplyUpdates('parentAccountId', parentAccountId);
 
     const completedAccountConfig = nextAccountConfig as FlatAccountFields_NEW;
-
+    if (!Object.hasOwn(this.config, 'accounts')) this.config.accounts = [];
```

Comment:
Another slight nit (and maybe just personal preference), but mixing the inline ifs with the bracketed ones makes this a little harder to read. I'd be fine with using brackets here to make it easier to follow, even if it adds a few lines.

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/172/commits/0559c584504dc458721cef1dced436eaf0e88862